### PR TITLE
Add temporary LIBRARY_INSTRUCTIONS for ART.

### PR DESCRIPTION
**Pull request by jkeck**
art-lib-instructions

### CODE REVIEW

File: lib/constants.rb [hunk 1]
```diff
@@ -1090,6 +1090,10 @@ module Constants
    }
    BROWSABLE_CALLNUMBERS = ["LC", "DEWEY", "ALPHANUM"]
    LIBRARY_INSTRUCTIONS = {
+     'ART' => {
+       heading: 'Library closed June25-August23',
```
Comment (mejackreed) [inline lib/constants.rb]: Should there be spaces here?

```
heading: 'Library closed June 25 - August 23',
```

Comment (jkeck) [inline lib/constants.rb]: @jvine?  I had the same thought, but just followed the image you sent me.

Comment (jvine) [inline lib/constants.rb]: Original text is what PB provided, but yeah, it's better w spaces. Let's do that.

Comment (jkeck) [inline lib/constants.rb]: done

**Pull request merged**